src/main/java/org/reaktivity/nukleus/http2/internal/routable/Correlation.java

```diff
@@ -20,24 +20,29 @@
 import java.util.Objects;
 import java.util.function.IntSupplier;
 
-import org.reaktivity.nukleus.http2.internal.routable.stream.PushHandler;
+import org.reaktivity.nukleus.http2.internal.routable.stream.IntObjectBiConsumer;
```

Comment:
Should be in `internal.util.function` package similar to `LongObjectBiConsumer` functional interface.